elements/lisk-chain/test/unit/block.spec.ts

```diff
@@ -53,5 +71,127 @@ describe('block', () => {
 				expect(() => block.validate()).not.toThrow();
 			});
 		});
+
+		describe('when an asset data has size more than the limit', () => {
+			it(`should throw error when asset data lenght is greater than ${MAX_ASSET_DATA_SIZE_BYTES}`, async () => {
+				// Arrange
+				const assets = [
+					{
+						moduleID: 3,
+						data: getRandomBytes(64),
+					},
+					{
+						moduleID: 4,
+						data: getRandomBytes(128),
+					},
+				];
+				block = await createValidDefaultBlock({ assets: new BlockAssets(assets) });
+				// Act & assert
+				expect(() => block.validate()).toThrow(
+					`Module with ID ${assets[1].moduleID} has data size more than ${MAX_ASSET_DATA_SIZE_BYTES} bytes.`,
+				);
+			});
+
+			it(`should pass when asset data lenght is equal or less than ${MAX_ASSET_DATA_SIZE_BYTES}`, async () => {
```

Comment:
typo at `lenght`

Reply:
seems like there were other places too where i did that typo 😅